frontend/src/views/ParticipantsList.js

```diff
@@ -0,0 +1,107 @@
+import React, { Fragment, useContext, useEffect } from "react"
+import { Link } from "react-router-dom"
+import { observer } from "mobx-react-lite"
+import { makeStyles } from "@material-ui/core/styles"
+import Table from "@material-ui/core/Table"
+// import TableRow from "@material-ui/core/TableRow"
+// import TableBody from "@material-ui/core/TableBody"
+// import TableCell from "@material-ui/core/TableCell"
+// import TableHead from "@material-ui/core/TableHead"
+import Breadcrumbs from "@material-ui/core/Breadcrumbs"
+import PersonAddIcon from "@material-ui/icons/PersonAdd"
+import BottomNavigation from "@material-ui/core/BottomNavigation"
+
+import PrevPointCopy from "../components/Typography/PrevPointCopy"
+// import PrevPointTitle from "../components/Typography/PrevPointTitle"
+import PrevPointHeading from "../components/Typography/PrevPointHeading"
+import { rootStoreContext } from "../stores/RootStore"
+import PrevPointTableHead from "./../components/ParticipantsListComponent/PrevPointTableHead"
```

Comment:
opinion: It would be nice to import these from a common module so they could be on one line e.g

`import { PrevPointTableHead, PrevPointTableBody } from "../components/prevpointtable"`

Reply:
That's fair and I can oblige. It does make more sense this way